vcd_cli/role.py

```diff
@@ -36,3 +41,30 @@ def list_roles(ctx):
         stdout(result, ctx)
     except Exception as e:
         stderr(e, ctx)
+
+
+@role.command('list_rights', short_help='list rights of a role')
```

Comment:
The sub command should include an hyphen `-` for consistency with other sub-commands: `vcd role list-rights`.